packages/frontend/package.json

```diff
@@ -20,21 +20,28 @@
   },
   "dependencies": {
     "@fnando/sparkline": "maciejhirsz/sparkline",
-    "@types/react-measure": "^2.0.5",
     "@polkadot/util-crypto": "^1.6.1",
+    "@types/react-measure": "^2.0.5",
+    "blakejs": "^1.1.0",
+    "husky": "^4.2.3",
+    "lint-staged": "^10.1.0",
+    "prettier": "^2.0.2",
     "react": "16.4.0",
     "react-dom": "16.4.2",
     "react-measure": "^2.3.0",
     "react-scripts-ts": "2.17.0",
     "react-svg": "^4.1.1",
     "stable": "^0.1.8",
-    "blakejs": "^1.1.0"
+    "tslint": "^6.1.0"
   },
   "scripts": {
+    "precommit": "lint-staged",
     "start": "react-scripts-ts start",
     "build": "react-scripts-ts build",
     "test": "echo 'FIXME: FE tests need updating'",
-    "eject": "react-scripts-ts eject"
+    "eject": "react-scripts-ts eject",
```

Comment:
I suggest you add a new `lint` script that does the magic. We can then call the linter with `yarn lint` in Travis before doing anything else.

Reply:
i wouldn't mess with this pr anymore, there are PRs that are related to this one it would mean changing them too. i can introduce that into the new one

Reply:
Since this PR is about prettifying and linting, we need a way to start checking whether linting is OK. This yarn script would be the place for that and we can add it to travis too.